posts/2022-01-18-beta-release-post.adoc

```diff
@@ -0,0 +1,146 @@
+
+An `X.509` certificate is a digital certificate which is used to manage identity and security in internet communications and computer networking.
+
+The `X.509` certificate can be sent by the intermediate servers (like web server, proxy server or load balancer) to an application server. Due to the deprecation of long lines folding in the `HTTP` header continuation, some of the open source intermediate servers start sending a client certificate in the Privacy-Enhanced Mail (`PEM`) url-encoded certificate to application server.
+
+Currently, WebSphere Application Server and Liberty Server do not support this new url-encoded certificate. Any request with this new format is rejected and the request is terminated.
+
+An example of a truncated url-encoded certificate sent to application server:app-name:
+
+[source]
+----
+[-----BEGIN%20CERTIFICATE-----%0AMIIDODCCAiCgAwIBAgIIHbO76YfCH1cwDQYJKoZIhvcNAQELBQAwOjELMAkGA1UE%0ABhMCVVMxDDAKBgNVBAoTA0lCTTEMMAoGA1UECxMDVFdTMQ8wDQYDVQQDEwZDbGll%0AbnQwHhcNMTUxMTAzMTYyODM1WhcNMzUxMDMwMTYyODM1WjA6MQswCQYDVQQGEwJV%0AUzEMMAoGA1UEChMDSUJNMQwwCgYDVQQLEwNUV1MxDz%2F%2Bpo%2FF%2Bzi%2F0sfAUwRdfgMm%2FAKAGzwMQufOYeKCgMULtq14QAJQnLmq4M%2FM00%0AC5QyeYtdaTou%2BMsLmoa1tkq2VSDVxAcktJyRSRsox36G7EHDLV4U2gtR6xczNEjw%0A2%2Bsj772FjdAMXRSR%0A-----END%20CERTIFICATE-----%0A]
+----
+
+Note: 
+* (1) The header (i.e BEGIN ) and footer (i.e END) information are appended before sending to application server.
+* (2) The encoded character (i.e %) presents in the certificate.
+
+The IBM application server code is updated to auto detect and process this new format.
+
+For more information, check out the link:https://github.com/OpenLiberty/open-liberty/issues/11680[GitHub issue] for this new feature.
+
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>22.0.0.2-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[22.0.0.2-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features
+
+As of 21.0.0.12 the Jakarta EE 9.1 features are now out of beta and fully supported.  That means that you can either use them in the official 22.0.0.1 release, or continue to use them in the beta 22.0.0.2-beta package.  Just as before, you can enable the individual features you want or you can just add the Jakarta EE 9.1 convenience feature to enable all of the Jakarta EE 9.1 beta features at once:
```

Comment:
Not sure I like this paragraph because there aren't any JEE 9.1 features in the beta zip. At least I would cut it down to the first sentence...
```suggestion
As of 21.0.0.12 the Jakarta EE 9.1 features are now out of beta and fully supported. [link to JEE 9.1 blog post]
```